docs/mp/oci/01_oci.adoc

```diff
@@ -0,0 +1,69 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2021 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+:javadoc-base-url-api: {javadoc-base-url}io.helidon.config/io/helidon/oci
+
+= Oracle Cloud Infrastructure Integration
+:h1Prefix: MP
+:description: Helidon OCI Integration
+:keywords: oci
+:common-deps-page-prefix-inc: ../../shared/dependencies/common_shared.adoc
+:feature-name: OCI Integration
+
+The Helidon MP OCI Integration provides easy access to Oracle Cloud Infrastructure.
+
+include::{common-deps-page-prefix-inc}[tag=maven-dependency]
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.helidon.integrations.oci</groupId>
+    <artifactId>helidon-integrations-oci-cdi</artifactId>
+</dependency>
+----
+
+== General configuration
+
+===  Using Helidon MP Properties configuration
+
+The first option to configure connection to OCI is to directly specify properties in `microprofile-config.properties` file:
+
+[source,properties]
+----
+oci.config.oci-profile.user=ocid1.user....
+oci.config.oci-profile.fingerprint=1c:6c:....
+oci.config.oci-profile.tenancy=ocid1.tenancy.oc1..
+oci.config.oci-profile.region=us-...
+oci.config.oci-profile.key-pem=<pem content>
+----
+
+=== Using OCI Configuration
+
+The second  option is via OCI configuration file. For authentication in OCI a special configuration file should be set up. The file is usually located at `~/.oci/config`
+
+[source,properties]
+----
+[DEFAULT]
+user=ocid1.user....
+fingerprint=1c:6c:....
+tenancy=ocid1.tenancy.oc1..
+region=us-...
+key_file=<path to key file>
+----
+
+More information how to set up on your environment: https://docs.oracle.com/en-us/iaas/Content/API/Concepts/sdkconfig.htm#SDK_and_CLI_Configuration_File[Official website]
```

Comment:
For more information on setting up basic configuration information, see [SDK and CLI Configuration File] in OCI Developer Resources. https://docs.oracle.com/en-us/iaas/Content/API/Concepts/sdkconfig.htm#SDK_and_CLI_Configuration_File

Reply:
fixed